routes/tasks.go

```diff
@@ -0,0 +1,167 @@
+package routes
+
+import (
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jinzhu/gorm"
+	"github.com/labstack/echo"
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/kirikami/go_exercise_api/database"
+)
+
+func SaveTaskHandler(c echo.Context) error {
+	const timeForm = "3 04 PM"
+	var ids int
+	var id int64
+
+	currentTime := time.Now()
+	tasks := []database.Task{}
+
+	db := c.Get("DBConnection").(*gorm.DB)
```

Comment:
@Kirikami Make `DB` as a field of `type ApiV1Handler` for now, no need to set DB into echo context. Later we will try to use `net/context` for "transferring" DB